packages/@webex/common/test/unit/spec/browser-detection.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import { getBrowserSerial } from '@webex/common';
+import window from 'global/window';
+import bowser from 'bowser';
+import {browserDetection} from '@webex/common/src/constants.js';
+
+describe('getBowserSerial()', () => {
+  it('should provide the parsed bowser user agent when available', () => {
+    //@ts-ignore
+    window.navigator = {userAgent: 'user agent data'};
+
+    const res = getBrowserSerial();
+    assert.deepEqual(res._ua, 'user agent data');
+  });
+
+  it('should provide an error message when the user agent is not available', () => {
+    //@ts-ignore
+    window.navigator = undefined;
+
+    const res = getBrowserSerial();
+    assert.deepEqual(res, {error: browserDetection.unableToAccessUserAgent});
+  });
+
+  it('should provide an error message object when bowser.getParser() fails', () => {
+    //@ts-ignore
+    window.navigator = {userAgent: 'user agent data'};
+
+    bowser.getParser = sinon.stub().throws(new Error('test error'));
+    const res = getBrowserSerial();
+    assert.deepEqual(res, {error: 'test error'});
+  });
+});
```

Comment:
New line at end of file, if you could amend this to prevent static analysis from throwing errors in CI/CD later.